examples/react-aria/src/components/DateField.css

```
@@ -2,6 +2,19 @@
 @import './Button.css';
 @import './theme.css';
 
+.file-input-control {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)